skills/prompt_versioning.md

```
@@ -0,0 +1,356 @@
+
+# Emergency rollback (instant)
+/prompt rollback orchestrate --emergency
+```
+
+---
+
+## Best Practices
+
+### 1. Semantic Versioning
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)